tests/test_datasets.py

```diff
@@ -25,24 +24,23 @@
 
 
 @pytest.fixture(scope="module")
-def setup_registry(make_test_project_dir_module, make_test_data_dir_module):
+def setup_registry(tmp_path_factory, make_test_project_dir_module, make_test_data_dir_module):
```

Comment:
This is an unrelated improvement. It uses a pytest fixture instead of making our own TemporaryDirectory.